### PR TITLE
Replace font loader

### PR DESCRIPTION
**Pull request by jamie-lynch**
## What are you doing in this PR?

This PR removes the `fontLoaderBundle` and replaces it with a `font.scss` file containing font-face declarations via the following steps:

* Add a new `fonts.scss` file containing font declarations in `support-frontend/assets/stylesheets/skeleton`
* Import this into `support-frontend/assets/stylesheets/skeleton/skeleton.scss` so that it is loaded (once) on the page
* Remove the `non-critical-webfonts.scss` file and import (as Titlepiece is included in the list of declarations)
* Remove the `font-loader.js` file from `support-frontend/assets/rendering/` 
* Remove reference to the `fontLoaderBundle` and wiring throughout the application

## Why are you doing this?

This change is being made as part of an effort to consolidate the usage of fonts across Guardian applications, including where fonts are loaded from and how they are loaded. 

cc @guardian/client-side-infra 

## Is this an AB test?
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by jamie-lynch** [comment]
AFAICT the `support-frontend-mono` build is failing due to `scalastyle` errors but I don't think they're related to any changes we've made. When I run it on main though, `scalastyle` is happy so now I'm confused. If anyone has got time to take a quick look, that would be much appreciated.

File: support-frontend/assets/stylesheets/gu-sass/typography.scss
Comment (i-hardy) [inline support-frontend/assets/stylesheets/gu-sass/typography.scss:13]: How would we replicate this in Emotion? (or is it handled by Source?) Sass is a legacy part of the codebase, it would be helpful to know how we migrate away from this in future.

**Comment by tomrf1** [comment]
> AFAICT the `support-frontend-mono` build is failing due to `scalastyle` errors but I don't think they're related to any changes we've made. When I run it on main though, `scalastyle` is happy so now I'm confused. If anyone has got time to take a quick look, that would be much appreciated.

Scalastyle was unhappy with the trailing commas, I've pushed commits to fix this and the tests.

File: support-frontend/assets/stylesheets/gu-sass/typography.scss
Comment (GHaberis) [reply support-frontend/assets/stylesheets/gu-sass/typography.scss:13]: @i-hardy I believe the typography exports from Source map to font-families that reference the newer font names as the primary font and use the old font name as a fallback if the new font in unavailable.

**Comment by jamie-lynch** [comment]
> > AFAICT the `support-frontend-mono` build is failing due to `scalastyle` errors but I don't think they're related to any changes we've made. When I run it on main though, `scalastyle` is happy so now I'm confused. If anyone has got time to take a quick look, that would be much appreciated.
> 
> Scalastyle was unhappy with the trailing commas, I've pushed commits to fix this and the tests.

Amazing, thanks Tom! 🙌

File: support-frontend/assets/stylesheets/gu-sass/typography.scss
Comment (jamie-lynch) [reply support-frontend/assets/stylesheets/gu-sass/typography.scss:13]: Here's a link to where Source sets the font values which backs up what George has said above: https://github.com/guardian/source/blob/main/src/core/foundations/src/theme.ts#L27

You can then use the Typography methods that it provides:
https://theguardian.design/2a1e5182b/p/95d5d0-code

Comment (sndrs) [reply support-frontend/assets/stylesheets/gu-sass/typography.scss:13]: these are [the font family name source uses](https://github.com/guardian/source/blob/0566752b3ce5851bc1a907db017e74819fcdcec1/src/core/foundations/src/theme.ts#L27-L35), so in theory you shouldn't need to do anything if you're using source.

cc @SiAdcock to make sure i've got that right!

**Comment by GHaberis** [comment]
Thanks for making these changes @jamie-lynch. This is a much simpler font-loading strategy, so should be much easier for anyone new to the codebase to understand.

**Comment by GHaberis** [comment]
The code looks good, although I did a quick search of the codebase and found a few hardcoded references amongst the styles to the old font names.

**Comment by jamie-lynch** [comment]
> The code looks good, although I did a quick search of the codebase and found a few additional hardcoded references amongst the styles to the old font names.

Ah yes! I've updated those now.

**Review by GHaberis — APPROVED** [review]
All looks good to me! 👍

**Comment by jamie-lynch** [comment]
Thanks for the review! I've added the `Do Not Merge` as, following a chat this morning, we want to make a couple of changes to the recommended declarations so it'd be good to get those in here before we merge.

**Comment by GHaberis** [comment]
> Thanks for the review! I've added the `Do Not Merge` as, following a chat this morning, we want to make a couple of changes to the recommended declarations so it'd be good to get those in here before we merge.

No problem! @jamie-lynch  I also think it'd be worth putting this up on our `CODE` environment for a quick check before releasing to `PROD`, @i-hardy should be available to coordinate this.

**Comment by sndrs** [comment]
i've updated the set of font-faces to match the recommended ones/source ones.

but agree about checking on CODE 👍

**Comment by GHaberis** [comment]
@sndrs @jamie-lynch can this be merged now?

**Comment by sndrs** [comment]
@GHaberis if your team is happy with it we are. I don't if it's been checked on code yet? i don't think we know how to do that...

**Pull request merged**
